### PR TITLE
Colormaps and some improvements in notebooks

### PR DESCRIPTION
**Pull request by mauro3**

### CODE REVIEW

File: app/forward_aletsch.jl [hunk 1]
```diff
@@ -34,41 +34,51 @@ In this notebook, we run the SIA model using the reconstructed distribution of t
...
+        		solve!(model)
+        		model.fields.H_old .= model.fields.H
+    		end
+
+    		# save the results
+    		V = Array(model.fields.V)
+    		H = Array(model.fields.H)
+
+			if flag==1
+				jldsave("../output/forward_aletsch/forward_aletsch_25m_snap_$(nt)yrs.jld2"; V, H)
```
Comment (luraess) [inline app/forward_aletsch.jl:68]:
```suggestion
			jldsave("../output/forward_aletsch/forward_aletsch_25m_snap_$(nt)yrs.jld2"; V, H)
```

File: app/forward_aletsch.jl [hunk 2]
```diff
@@ -34,41 +34,51 @@ In this notebook, we run the SIA model using the reconstructed distribution of t
...
+			else
+    			jldsave("../output/forward_aletsch/forward_aletsch_25m_time_dep_$(nt)yrs.jld2"; V, H)
```
Comment (luraess) [inline app/forward_aletsch.jl:70]:
```suggestion
    			    jldsave("../output/forward_aletsch/forward_aletsch_25m_time_dep_$(nt)yrs.jld2"; V, H)
```

File: app/forward_aletsch.jl (hunk 1 above)
Comment (luraess) [inline app/forward_aletsch.jl:68]:
```suggestion
			    jldsave("../output/forward_aletsch/forward_aletsch_25m_snap_$(nt)yrs.jld2"; V, H)
```

File: app/make_figures.jl [hunk 3]
```diff
@@ -788,6 +962,124 @@ md"""
 ## Comparing Aletsch runs forward in time using snapshot and time-dependent inputs of basal sliding
 """
 
+# ╔═╡ 1315f741-afa2-4eb5-9ef6-243115ec8746
```
Comment (mauro3) [inline app/make_figures.jl:965]: @luraess : this is the new fig 7. Maybe you can double check, together with forward_aletsch.jl https://github.com/yiluchen1066/Glaide.jl/pull/7/files#diff-53ecee046e1cdc1f156ba3549f69be8edfbd47f15579fa4770f15d70a2a760bb

**Comment by luraess** [comment]
Can we merge this?

**Pull request merged**